### DISCUSSION TITLE
how are you playing sound? without offscreen api?

### DISCUSSION THREAD

**Discussion by rahulraikwar00** [schmich/marinara#526] [Q&A]
same here how are you playing sound? without offscreen api?

**Comment by schmich** [comment]
It's nothing fancy. I'm using the [AudioContext API](https://developer.mozilla.org/en-US/docs/Web/API/AudioContext) in a background script. As far as I remember, no special configuration or permissions are needed to do this. See [the source](https://github.com/schmich/marinara/blob/master/src/Sounds.js#L63) for details.

**Comment by rahulraikwar00** [comment]
yh i saw it already ..  do you have plans to move to V3? its bit tricky to play audio there we have to create offscreen document for that .. AudioContextAPI don't work in  manifest V3
also if you can drop any advance resource on Chrome extension i would really appreciate that..... thank you
